pos_invoice_send_mail/models/pos_config.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Druidoo - Iván Todorovich
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    iface_invoice_mail = fields.Boolean('Send Invoice by Mail')
+
+    invoice_mail_template_id = fields.Many2one(
+        'mail.template',
+        string='Invoice Email Template',
+        domain=[('model', '=', 'account.invoice')],
+        context=lambda self: {
```

Comment:
Why this line. Is not default value enough ?

Reply:
It's only for usability.
If the user tries to create a new email template using "Create and edit..", it'll have the correct model set by default.